### PR TITLE
S3 key prefix

### PR DESCRIPTION
**Pull request by leonidcliqz**
Added a prefix flag for the S3 path

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -16,8 +16,10 @@ services:
       BASE_URL: /
       DB_PASSWORD: mypassword
       AWS_DYNAMODB_TABLE: ${AWS_DYNAMODB_TABLE}
+      AWS_KEY_PREFIX: ${AWS_KEY_PREFIX}
+      TERRABOARD_LOG_LEVEL: ${TERRABOARD_LOG_LEVEL}
     links:
-      - db
```
Comment (raphink) [inline docker-compose.yml]: This doesn't seem linked to the PR.

**Comment by raphink** [comment]
Thanks for the PR. It looks good, expect for the change in `docker-compose.yml`, which doesn't seem to belong there.

**Comment by leonidcliqz** [comment]
Basically, it wasn't working without it :)

**Comment by raphink** [comment]
I'm surprised it actually works with it, given that the default value for `Config.DB.Host` is `db`, not `database`. Do you mind opening another PR for that change?

**Comment by raphink** [comment]
Maybe your local DNS resolves `db`?

**Comment by leonidcliqz** [comment]
Nope, but now it started to work. Sorry for the mess

**Comment by raphink** [comment]
No problem :smile: If you remove the line, I'll merge the PR.

**Pull request merged**